packages/hive/sodasql/dialects/hive_dialect.py

```diff
@@ -58,6 +59,29 @@ def create_connection(self, *args, **kwargs):
         except Exception as e:
             self.try_to_raise_soda_sql_exception(e)
 
+    def __query_table(self, table_name):
+        query = f"""
+        SELECT *
+        FROM {table_name}
+        LIMIT 1
+        """
+        return query
+
+    def sql_test_connection(self) -> Union[Exception, bool]:
```

Comment:
The return type doesn't look like what the function does, raising exceptions don't return them. So maybe you mean just bool here?

Reply:
This applies to other dialects too, btw :)

Reply:
Some more context: https://www.python.org/dev/peps/pep-0484/#exceptions

Reply:
right, it's either we return boolean or we raise an exception, so bool is enough!